### PR TITLE
Fix scan-build require line

### PR DESCRIPTION
**Pull request by serge-sans-paille**

### CODE REVIEW

**Comment by sylvestre** [comment]
```
    LLVM regression suite :: scanbuild_missing_delete.cpp
```
is that expected?

**Comment by serge-sans-paille** [comment]
no, that's what I'm trying to fix :-)

**Comment by serge-sans-paille** [comment]
no, that's what I'm trying to fix :-)

**Comment by serge-sans-paille** [comment]
Finally works, yay!

File: tests/scanbuild_missing_delete.cpp [hunk 1]
```diff
@@ -1,8 +1,10 @@
 // Tests scan-view runs correctly, see https://bugzilla.redhat.com/show_bug.cgi?id=1647130
-// REQUIRES: scanbuild, scanview
-// RUN: mkdir %t.out
-// RUN: %scan-build -o %t.out %clangxx -c %s -o %t
-// RUN: timeout 10 %scan-view --no-browser %t.out/*
+// REQUIRES: scan-build, scan-view
+// RUN: mkdir %t.out && cd %t.out && rm -rf *
+// RUN: %scan-build -o . %clangxx -c %s -o %t
+// It's not super satisfying to call sh like this, and not portable on windows.
+// Yet that's the only I found to cope with the lack of support for '&' in lit.
```
Comment (sylvestre) [inline tests/scanbuild_missing_delete.cpp:6]: ugly indeed :)

**Pull request merged**